### PR TITLE
Fix pybel import from openbabel.

### PR DESCRIPTION
**Pull request by csadorf**

### CODE REVIEW

File: aiidalab_widgets_base/structures.py [hunk 1]
```diff
@@ -555,7 +555,7 @@ def _optimize_mol(self, mol):
 
         # Note, the pybel module imported below comes together with openbabel package. Do not confuse it with
         # pybel package available on PyPi: https://pypi.org/project/pybel/
-        import pybel  # pylint:disable=import-error
+        from openbabel import pybel  # pylint:disable=import-error
```
Comment (yakutovicha) [inline aiidalab_widgets_base/structures.py:558]:
```suggestion
        try:
            import pybel  # pylint:disable=import-error
        except ModuleNotFoundError:
            from openbabel import pybel  # pylint:disable=import-error
```

File: aiidalab_widgets_base/structures.py [hunk 2]
```diff
@@ -578,7 +578,7 @@ def _on_button_pressed(self, change):  # pylint: disable=unused-argument
...
```
Comment (yakutovicha) [inline aiidalab_widgets_base/structures.py:581]:
```suggestion
        try:
            import pybel  # pylint:disable=import-error
        except ModuleNotFoundError:
            from openbabel import pybel  # pylint:disable=import-error
```

**Review by yakutovicha — COMMENTED** [review]
thanks, @csadorf for catching this.

As we cannot control the open babel version which is installed through conda, maybe use try-except (see suggestions).

**Comment by csadorf** [comment]
> As we cannot control the open babel version which is installed through conda, maybe use try-except (see suggestions).

I am not necessarily against that, but why do you say that we cannot control that?

https://github.com/aiidalab/aiidalab-docker-stack/blob/a7a0b9e4077f7c8f66cc25ec1d2e46cefa852a2f/Dockerfile#L103

**Comment by yakutovicha** [comment]
as we discussed, the issue has been resolved in #158

**Pull request closed without merging**